src/schemas.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { z } from 'zod';
+import {
+  IAddressMetadataAsRecord,
+  IAuthoritiesBalance,
+  IBalance,
+  IHistoryInput,
+  IHistoryOutput,
+  IHistoryOutputDecoded,
+  IHistoryTx,
+  ILockedUtxo,
+  ITokenBalance,
+  ITokenMetadata,
+  IUtxo,
+  TxHistoryProcessingStatus,
+} from './types';
+import { ZodSchema } from './utils/bigint';
+
+export const ITokenBalanceSchema: ZodSchema<ITokenBalance> = z
```

Comment:
`ITokenBalanceSchema` starts with `I` because it is an interface (just a naming convention).
Maybe we should change the names since we are using zod schemas for validation/typing

Reply:
I kept it purposefully to indicate that this is schema is tied to an existing interface, in this case, `ITokenBalance`. So this is `ITokenBalance`'s schema, or, `ITokenBalanceSchema`.

I do agree this could be improved and the `I` becomes unnecessary when we remove the interfaces and use `Zod.infer` instead. This is set to be changed in this issue: https://github.com/HathorNetwork/hathor-wallet-lib/issues/800, so I would rather do it as part of another PR. What do you think?

Reply:
Great!
Just wanted to keep consistency to avoid any confusion later